be/src/exec/workgroup/work_group.h

```diff
@@ -148,13 +150,16 @@ class WorkGroup {
     static constexpr int64 DEFAULT_WG_ID = 0;
     static constexpr int64 DEFAULT_VERSION = 0;
 
+    int64_t mem_limit() const;
+
 private:
     std::string _name;
     int64_t _id;
     int64_t _version;
 
     size_t _cpu_limit;
     double _memory_limit;
+    int64_t _mem_limit = -1;
```

Comment:
_memory_limit and _mem_limit are confusing

Reply:
Yes may be a little confusing,  i will change the `_mem_limit` to `_memory_limit_bytes`